ui/src/Manager.cs

```diff
@@ -233,5 +245,57 @@ public static void InitializeSettings()
         {
             Settings = new Settings(ProductConstants.SettingsFile);
         }
+
+        /// <summary>
+        /// Initializes cache for avatar image.
+        /// </summary>
+        public static void InitializeCache()
```

Comment:
If you believe there should be a need for more caching later, remove "for avatar image" from function description.

If not, rename `InitializeCache()` to `InitializeAvatarCache()` or whatever you think is best.